ql/termstructures/yield/fittedbonddiscountcurve.cpp

```diff
@@ -248,7 +248,10 @@ namespace QuantLib {
 
     FittedBondDiscountCurve::FittingMethod::FittingCost::FittingCost(
                         FittedBondDiscountCurve::FittingMethod* fittingMethod)
-    : fittingMethod_(fittingMethod) {}
+    : fittingMethod_(fittingMethod) {
+        for (Size i = 0; i < fittingMethod_->curve_->bondHelpers_.size(); ++i)
```

Comment:
Maybe part of `FittingMethod::init` instead? Here it seems kind of hidden away to me...

Reply:
Yes. And I just noticed we can remove some more code there as well.